docs/MultipleLoadNativeModule.md

```diff
@@ -0,0 +1,65 @@
+# Load Multiple Instances of Native Module
+
+When using native mode on Windows, the programs are loaded from a native driver instead of an ELF file. As part of the load operation, the native driver is loaded in the kernel and the eBPF execution context orchestrates the creation of maps and loading the programs from the native driver. The execution context fetches the map properties from the native module (using functions exported by the native driver via NMR) and creates the required maps.
```

Comment:
https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#formatting-notes says:
> Source lines **MUST NOT** exceed 120 columns.

Reply:
Fixed throughout the doc.